<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RangeStateWindow</class>
 <widget class="QDialog" name="RangeStateWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>938</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Range State Window</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,2,0,2">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Candidate Signals:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listCandidates"/>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Graphical Representation of Selected Signal:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCustomPlot" name="graphSignal" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Sensitivity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="slideSensitivity">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>250</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Min Signal Size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinMinSigSize">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>32</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Max Signal Size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinMaxSigSize">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>32</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Granularity:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinGranularity">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>32</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Signal Mode:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbSignalMode"/>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Signed Mode:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbSignedMode"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>ID Filter:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listFilter"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnAllFilter">
         <property name="text">
          <string>All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnNoneFilter">
         <property name="text">
          <string>None</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnRecalc">
     <property name="text">
      <string>Recalculate Candidate Signals</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>slideSensitivity</tabstop>
  <tabstop>spinMinSigSize</tabstop>
  <tabstop>spinMaxSigSize</tabstop>
  <tabstop>spinGranularity</tabstop>
  <tabstop>cbSignalMode</tabstop>
  <tabstop>cbSignedMode</tabstop>
  <tabstop>listFilter</tabstop>
  <tabstop>btnAllFilter</tabstop>
  <tabstop>btnNoneFilter</tabstop>
  <tabstop>btnRecalc</tabstop>
  <tabstop>listCandidates</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
